test/libsolidity/semanticTests/reverts/error_struct.sol

```diff
@@ -1,18 +0,0 @@
-struct error { uint error; }
```

Comment:
Why wasn't this also moved to syntax tests?

Reply:
We already have two tests for parser errors for `error`, and this one doesn't really add anything more, I think.